{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -0,0 +1,158 @@
+kind: live
+title: {{ cookiecutter.project_name }}
+id: PROJECT_SLUG_WITH_UNDERSCORES_PLACEHOLDER
+
+defaults:
+  preset: gpu-small-p
+  life_span: 1d
+  tags:
+    - kind:flow
+    - flow-id-with-dashes:{{ cookiecutter.project_slug }}
```

Comment:
What is the purpose of these custom tags?

Reply:
well, we want to include flow-id here, but we can't use `flow-id` as it contains underscores what are forbidden in job tags.

I'm ready to drop this tag at all.

Reply:
neuro-flow uses system tags to pin project, flow, and job ids to the executed job.
They are named like `project:{...}`, `flow:{...}` and `job:{...}`.
I suggest just keeping them

Reply:
:+1: removing it at all